frontend/src/components/housing/project/ProjectForm.vue

```diff
@@ -0,0 +1,1069 @@
+    } else if (atsCreateType.value === ATSCreateTypes.ENQUIRY) {
+      values.atsEnquiryId = await createATSEnquiry();
+      if (values.atsEnquiryId) {
+        values.addedToAts = true;
+      }
+      atsCreateType.value = undefined;
+    }
+
+    // Grab the contact information
+    const contact = {
+      contactId: values.contact.contactId,
+      firstName: values.contact.firstName,
+      lastName: values.contact.lastName,
+      phoneNumber: values.contact.phoneNumber,
+      email: values.contact.email,
+      contactApplicantRelationship: values.contact.contactApplicantRelationship,
+      contactPreference: values.contact.contactPreference
+    };
+
+    // Generate final submission object
+    const dataOmitted = omit(
+      setEmptyStringsToNull({
+        ...values.project,
+        ...values.units,
+        ...values.location,
+        ...values.finance,
+        ...values.submissionState,
+        activityId: project.activityId,
+        housingProjectId: project.housingProjectId,
+        projectDescription: values.projectDescription,
+        projectLocationDescription: values.projectLocationDescription,
+        astNotes: values.astNotes,
+        atsClientId: parseInt(values.atsClientId) || '',
+        atsEnquiryId: parseInt(values.atsEnquiryId) || '',
+        aaiUpdated: values.aaiUpdated,
+        addedToAts: values.addedToAts,
+        ltsaCompleted: values.ltsaCompleted,
+        bcOnlineCompleted: values.bcOnlineCompleted,
+        submittedAt: values.submittedAt,
+        consentToFeedback: values.consentToFeedback === BasicResponse.YES,
+        naturalDisaster: values.location.naturalDisaster === BasicResponse.YES,
+        assignedUserId: values.submissionState.assignedUser?.userId ?? undefined
+      }),
+      [
+        'contactId',
+        'contactFirstName',
+        'contactLastName',
+        'contactPhoneNumber',
+        'contactEmail',
+        'contactApplicantRelationship',
+        'contactPreference',
+        'contactUserId',
+        'assignedUser',
+        'submissionState',
+        'locationAddress',
+        'relatedEnquiries'
+      ]
+    );
+
+    // Update project - order of calls is important
+    const contactResponse = (await contactService.updateContact(contact)).data;
+    await activityContactService.updateActivityContact(project.activityId, [
+      { ...contact, contactId: contactResponse.contactId }
+    ]);
+    const result = await housingProjectService.updateProject(project.housingProjectId, dataOmitted);
+
+    // Update store with returned data
+    projectStore.setProject(result.data);
+
+    // Reinitialize the form
+    formRef.value?.resetForm({
+      values: {
+        ...initilizeFormValues(result.data)
+      }
+    });
+
+    toast.success(t('i.common.form.savedMessage'));
+  } catch (e: any) {
+    toast.error(t('i.common.projectForm.failedMessage'), e.message);
+  }
+};
+
+function updateLocationAddress(values: any, setFieldValue?: Function) {
```

Comment:
Something not working here, location isn't being updated on changes... may be another `setFieldValue` issue.